modules/nodes-nodes-cgroups-2.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="nodes-nodes-cgroups-2_{context}"]
+= Enabling cgroups version 2
+
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster by using a machine config. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies.
+
+[NOTE]
```

Comment:
Should this be an IMPORTANT? 
Similar to this:
https://docs.openshift.com/container-platform/4.9/scalability_and_performance/ztp-deploying-disconnected.html#configuring_bios_for_du_hosts_ztp-deploying-disconnected